docs/training/cult-app-pocket

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Pocket Cult Application
+parent: Cult
+grand_parent: Training
+nav_order: 2
+---
+
+# ❋ Chaos Indivisvm Application (P) v.1.00❋
+{: .no_toc }
+
+Chaos Indivisvm is located at **Milopolis**
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+# The Application
+
+Hello! This is the Chaos Indivisvm POCKET APPLICATION. If you choose to fill this form out, 
+you will b interviewed before you are considered to join Chaos Indivisvm's cult.
+
+Your interview will require a microphone and will be conducted via Second Life Voice, Discord, or Mumble based on the decision of your interviewer.
```

Comment:
interview will be done in IM so I still have something i can copy/paste into their thread on the forums.

Reply:
added part, fixed